docs/scenes/disney_blend_creation/.ipynb_checkpoints/disney_blend_scene_creation-checkpoint.py

```diff
@@ -0,0 +1,228 @@
+import enoki as ek 
+import mitsuba
+mitsuba.set_variant('scalar_rgb')
+import numpy as np
+import torch
+from mitsuba.core import Bitmap, Struct
+import matplotlib.pyplot as plt
+from mitsuba.python.util import traverse
+from mitsuba.core import load_file
+
+def create_images(scene,params,parameter_changed):
+    images=[]
+    for j in range(11):
+        params['Material.'+parameter_changed] = j/10
+        params.update()
+        sensorIndex=0
+        image= scene.integrator().render(scene,seed=1, sensor_index=sensorIndex);
+        # Get the scene's sensor (if many, can pick one by specifying the index)
+        sensor = scene.sensors()[sensorIndex];
+        # The rendered data is stored in the film
+        film = sensor.film();
+        img= film.bitmap();
```

Comment:
Hi @efylmzr ,

As for the other PR, it would be great if you could run the `autopep8` python formatter to avoid those kind of typos 😉